app/domain/authentication/authn_azure/validate_resource_restrictions.rb

```diff
@@ -36,7 +36,7 @@ def validate_resource_restrictions_match_request
         resource_restrictions.resources.each do |resource_from_role|
           resource_from_request = resources_from_request.find { |resource| resource == resource_from_role }
           unless resource_from_request
-            raise Errors::Authentication::Jwt::InvalidResourceRestrictions, resource_from_role.type
+            raise Errors::Authentication::InvalidResourceRestrictions, resource_from_role.type
```

Comment:
do we want a `ResourceRestrictions` module?

Reply:
@Tovli  commented to me (offline) that he thinks `Common` is a bad module name, as it ends as a garbage can for anything that it is not obvious where it belongs.
I'm all in for better name, though.
What do you have in mind?

Reply:
i wasn't suggesting a `Common` module :)

Reply:
I want a module to group these classes.
This name seems more specific for its internals, so I think it is a good name.

Reply:
what am i missing? the object is `Errors::Authentication::InvalidResourceRestrictions` so there's no specific module.

Reply:
done